content/200-concepts/200-database-connectors/08-cockroachdb.mdx

```diff
@@ -0,0 +1,71 @@
+| `xml`                         | `String`  |  Not yet  | `@db.Xml`                      | Supported in PostgreSQL but not currently in CockroachDB |
+
+## Other limitations
+
+The following table lists any other limitations of CockroachDB compared to PostgreSQL:
+
+| Issue                                                        | Area   | Notes                                                                                                                                                                                                                                                  |
+| ------------------------------------------------------------ | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| `Json[]` type not supported                                  | Schema | `Json` arrays are currently not supported in CockroachDB                                                                                                                                                                                               |
+| `@@index(type: Hash)` not supported                          | Schema | In PostgreSQL, Prisma allows [configuration of indexes](/concepts/components/prisma-schema/indexes#configuring-the-access-type-of-indexes-with-type-postgresql) to use the `Hash` index access method. This is currently not supported in CockroachDB. |
+| Pushing to `Enum` types not supported                        | Client | Pushing to `Enum` types (e.g. `data: { enum { push: "A" }, }`) is currently not supported in CockroachDB.)                                                                                                                                             |
+| Searching on String fields without full text index...        | Client | Searching on String fields without full text index...                                                                                                                                                                                                  |
+| Integer division...                                          | Client | Integer division...                                                                                                                                                                                                                                    |
+| Only supports `equals` and `not` filtering on Json fields... | Client | Only supports `equals` and `not` filtering on Json fields...                                                                                                                                                                                           |
```

Comment:
Here support is just limited, not completely removed. In PostgreSQL this also supports other terms.

Reply:
Done